### DISCUSSION TITLE
Uno sample not working very well

### DISCUSSION THREAD

**Discussion by Marc-Antoine-Soucy** [Mapsui/Mapsui#2174] [Q&A]
[On this sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MapControlSample), there are several problems, and I'm not sure if it's supposed to work or not? [I opened an issue on the uno.sample repository](https://github.com/unoplatform/Uno.Samples/issues/466). The problem is that the slider that handles the zoom does not track when we zoom with something else than the slider.  Also, the slider straight up doesnt work on android, for example. I looked at the Mapsui documentation a bit, and I'm wondering if this is a bug from MapUI, or if it's a problem with the sample itself?

**Comment by pauldendulk** [comment]
Mapsui does not include a zoom slider control so it is probably a problem with the sample.

About the slider not updating when zooming in some other way. If you want this to work you would have to listen to the Navigator.ViewportChanged and update the slider for the new zoom level. I except they just did not implement this in that sample. 

About it not working on Android. I do not expect a difference there, so it is curious, and I would like to know what causes this. Still it might be simply a matter of calling RefreshGraphics to fix it. What happens if you start panning on Android? Does it jump to the expected zoom level? Or does it stay on the same level? Does it respond at all?

**Comment by inforithmics** [comment]
Pull Request fixing the sample
https://github.com/unoplatform/Uno.Samples/pull/581
